bundles/org.openhab.binding.chatgpt/README.md

```diff
@@ -22,6 +29,10 @@ API keys can be created and managed under <https://platform.openai.com/account/a
 
 The advanced parameters `apiUrl` and `modelUrl` can be used, if any other ChatGPT-compatible service is used, e.g. a local installation of [LocalAI](https://github.com/go-skynet/LocalAI).
 
+## Items Configuration
+
+You will need to tag [ "OpenAI" ] items which you want to control via chatGPT. Item names should look like Location_Type, for example "Kitchen_Light". In label describe item in more detail, if necessary, describe what commands it accepts.
```

Comment:
Probably because I did it for myself :) I don't use semantic model at all. No problem, I can do it with semantic model. But I would like to keep this "name method" as an additional option for people who don't use semantic model. About metadata - I found it difficult to get this data. Another registry, parsing, etc. But you are right, storing it in the label was a bad idea. I will fix it.

Reply:
Fair enough. If you keep the "item name approach" as an advanced configuration parameter, it should be fine.
But what about the "OpenAI" tag? I would vote to remove this as well, because the tags are meant to be used for the semantic model; if you want to tag certain items to be relevant for GPT, this should rely on a certain metadata as well.